internal/standalone_utils.go

```diff
@@ -0,0 +1,45 @@
+// Copyright © 2024 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package internal
+
+import (
+	"google.golang.org/grpc"
+)
+
+const DefaultStandaloneGRPCTarget = "localhost:50051"
+
+var StandaloneConnectorUtilities []ConnectorUtilityService
+
+type ConnectorUtilityService interface {
```

Comment:
Naming nitpick, the global var and interface name don't exactly match, even though they are closely related. I'd rename them so they match, for instance:
```suggestion
var StandaloneConnectorUtilities []StandaloneConnectorUtility

type StandaloneConnectorUtility interface {
```

Reply:
Good point.